components/head.tsx

```diff
@@ -19,6 +19,14 @@ export default function Head({
       <meta property="og:image" content="/logo_t.svg" />
       <meta property="og:url" content="https://the-algorithms.com" />
       <meta property="og:type" content="website" />
+      <meta property="og:locale" content="en_US" />
+      <meta property="og:locale:alternate" content="de_DE" />
+      <meta property="og:locale:alternate" content="es_MX" />
+      <meta property="og:locale:alternate" content="eo" />
+      <meta property="og:locale:alternate" content="uk" />
+      <meta property="og:locale:alternate" content="hr" />
+      <meta property="og:locale:alternate" content="zh_Hans" />
+      <meta property="og:locale:alternate" content="it" />
```

Comment:
I would suggest generating the locales automatically, this way they don't have to be updated.

```suggestion
      {locales.map((locale) => (
        <meta
          property={
            router.locale === locale ? "og:locale" : "og:locale:alternative"
          }
          content={locale}
          key={locale}
        />
      ))}
```

You would also have to add `import locales from "lib/locales";` at the top of the document and `const router = useRouter();` in the component funtion.